content/en/blog/_posts/2023-11-14-PodReadyToStartContainersCondition-in-beta.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: blog
+title: "Kubernetes 1.29: PodReadyToStartContainersCondition in beta"
+date: 2023-11-13
+slug: pod-ready-to-start-containers-condition-now-in-beta
+---
+
+**Author**: zefeng chen, Kevin Hannon(Red Hat)
+
+
+With recent release of Kubernetes 1.29 the `PodReadyToStartContainers`[condition](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions) is available by default.
+The kubelet manages the value for that condition throughout a Pod's lifecycle, 
+in the status field of a pod. The kubelet will use the PodReadyToStartContainers 
+condition to accurately surface the initialization state of a pod,
+From the perspective of pod sandbox creation and network configuration by a container runtime
+
+## What's the motivation and why we need it?
+
+Cluster administrators currently do not have a clear and easily accessible way to view the status of successfully scheduled pods 
```

Comment:
I would probably drop the successfully scheduled pods and say something like:

Cluster administrators currently do not have a clear and easily accessible way to view the completion of pod sandbox creation.

Reply:
updated